board/safety/safety_honda.h

```diff
@@ -440,7 +440,8 @@ static int honda_bosch_fwd_hook(int bus_num, CANPacket_t *to_fwd) {
   if (bus_num == 2)  {
     int addr = GET_ADDR(to_fwd);
     int is_lkas_msg = (addr == 0xE4) || (addr == 0xE5) || (addr == 0x33D) || (addr == 0x33DA) || (addr == 0x33DB);
-    if (!is_lkas_msg) {
+    int is_acc_msg = honda_bosch_radarless && (addr == 0x1C8);
+    if (!is_lkas_msg && !is_acc_msg) {
```

Comment:
Then need to check the longitudinal flag here to conditionally block the ACC message.